tests/de/media_player_HassMediaPause.yaml

```diff
@@ -14,3 +14,26 @@ tests:
       slots:
         name: "TV"
     response: "Pausiert"
+  - sentences:
+      - "pausiere"
```

Comment:
we should have at least one test sentence for each possible permutation of `(pausiere[n]|pause|anhalten|stopp|halt[e an])`

Reply:
definitely, commit coming